README.md

```diff
@@ -128,17 +148,226 @@ To start Parity Ethereum as a regular user using `systemd` init:
+    ```bash
+    ethcore-light
+    ```
+  * Parity Smart Contract based Node Filter, Manage Permissions of Network Connections
+    ```bash
+    node-filter
+    ```
+  * Parity Private Transactions
+    ```bash
+    ethcore-private-tx
+    ```
+  * Parity Ethereum (EthCore) Client & Network Service Creation & Registration with the I/O Subsystem
+    ```bash
+    ethcore-service
+    ```
+  * Parity Ethereum (EthCore) Blockchain Synchronization
+    ```bash
+    ethcore-sync
+    ```
+  * Parity Ethereum Common Types
+    ```bash
+    common-types
+    ```
+  * Parity Ethereum Virtual Machines (VM) Support Library
+    ```bash
+    vm
+    ```
+  * Parity Ethereum WASM Interpreter
+    ```bash
+    wasm
+    ```
+  * Parity Ethereum WASM Test Run
+    ```bash
+    pwasm-run-test
+    ```
+  * Parity EVM Implementation
+    ```bash
+    evmbin
+    ```
+  * Parity Ethereum IPFS-compatible API
+    ```bash
+    parity-ipfs-api
+    ```
+  * Parity Ethereum JSON Deserialization
+    ```bash
+    ethjson
+    ```
+  * Parity Ethereum State Machine Generalization for Consensus Engines
+    ```bash
+    parity-machine
+    ```
+* Parity Ethereum (EthCore) Miner Interface
+  ```bash
+  ethcore-miner parity-local-store price-info ethcore-stratum using_queue
+  ```
+* Parity Ethereum (EthCore) Logger Implementation
+  ```bash
+  ethcore-logger
+  ```
+* C bindings library for the Parity Ethereum client
+  ```bash
+  parity-clib
+  ```
+* Parity Ethereum JSON-RPC Servers
+  ```bash
+  parity-rpc
+  ```
+* Parity Ethereum (EthCore) Secret Store
+  ```bash
+  ethcore-secretstore
+  ```
+* Parity Updater Service
+  ```bash
+  parity-updater parity-hash-fetch
+  ```
+* Parity Core Libraries (Parity Util)
+  ```bash
+  ethcore-bloom-journal blooms-db dir eip-712 fake-fetch fastmap fetch ethcore-io
+  journaldb keccak-hasher len-caching-lock macros memory-cache memzero
+  migration-rocksdb ethcore-network ethcore-network-devp2p panic_hook
+  patricia-trie-ethereum registrar rlp_compress rlp_derive parity-runtime stats
+  time-utils triehash-ethereum unexpected parity-version
+  ```
+* Parity Whisper Protocol Implementation
+  ```bash
+  parity-whisper whisper-cli
+  ```
+
+</p></details>
+
+### Contributing to documentation for Parity Ethereum packages
+
+[Document source code](https://doc.rust-lang.org/1.9.0/book/documentation.html) for Parity Ethereum packages by annotating the source code with documentation comments.
+
+Example (generic):
+```markdown
+/// Summary
+///
+/// Description
+///
+/// # Panics
+///
+/// # Errors
+///
+/// # Safety
+///
+/// # Examples
+///
+/// Summary of Example 1
+///
+/// ```rust
+/// // insert example 1 code here
+/// ```
+///
+```
+
+* Important notes:
+  * Modules are documented using `//!`
+    ```
+    //! Summary (of module)
+    //!
+    //! Description (of module)
+    ```
+  * Special section header is indicated with a hash `#`.
+    * `Panics` section requires an explanation if the function triggers a panic
```

Comment:
I suspect that a `Panics` section in the module docs would be fairly rare (maybe discussing general panic behaviour in the module). Panic sections are common (and highly useful) in function/method docs.

Reply:
Thanks. I've pushed commit 805e39db2 that removes this. The user can refer to the Rust docs for information anyway